python_modules/libraries/dagster-docker/dagster_docker/docker_run_launcher.py

```diff
@@ -64,13 +66,22 @@ def config_type(cls):
     def from_config_value(inst_data, config_value):
         return DockerRunLauncher(inst_data=inst_data, **config_value)
 
-    def _get_client(self):
+    @property
+    def instance_container_context(self) -> DockerContainerContext:
+        return DockerContainerContext(
+            self.registry, self.env_vars, self.networks, self.container_kwargs
+        )
```

Comment:
whats this? the `instance_` prefix is confusing given conceptual gravity of DagsterInstance